app/models/user.rb

```diff
@@ -262,6 +262,17 @@ def can_read?(notebook, use_admin=false)
       (use_admin && admin?)
   end
 
+  def owner(notebook)
```

Comment:
You wrote this function...but then don't use it in the verify_owner function?

Reply:
I used it for the view. I guess I forgot I might be able to use it for the controller. Will play with it in a few

Reply:
Got it now. Works perfectly. Good eye